Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -54,41 +62,44 @@ final class PledgeCTAContainerView: UIView {
   override func bindStyles() {
     super.bindStyles()
 
-    _ = self.pledgeCTAbutton
-      |> projectStateButtonStyle
+    let isAccessibilityCategory = self.traitCollection.preferredContentSizeCategory.isAccessibilityCategory
 
-    _ = self.youreABackerLabel
-      |> \.font .~ .ksr_headline(size: 14)
-      |> \.text %~ { _ in Strings.Youre_a_backer() }
+    _ = self.amountAndRewardTitleStackView
+      |> \.axis .~ NSLayoutConstraint.Axis.vertical
+      |> \.isLayoutMarginsRelativeArrangement .~ true
 
     _ = self.amountOrRewardLabel
-      |> \.font .~ .ksr_caption1(size: 14)
+      |> \.font .~ .ksr_headline(size: 14)
       |> \.textColor .~ .ksr_dark_grey_500
+      |> \.numberOfLines .~ 0
 
-    _ = self.rootStackView
-      |> checkoutAdaptableStackViewStyle(
-        self.traitCollection.preferredContentSizeCategory.isAccessibilityCategory
+    _ = self.pledgeButton
+      |> projectStateButtonStyle(
+        isAccessibilityCategory,
+        amountAndRewardTitleStackViewIsHidden: self.amountAndRewardTitleStackView.isHidden
       )
-      |> \.distribution .~ UIStackView.Distribution.equalCentering
-      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+
+    _ = self.rootStackView
+      |> adaptableStackViewStyle(isAccessibilityCategory)
       |> \.isLayoutMarginsRelativeArrangement .~ true
       |> \.layoutMargins .~ UIEdgeInsets.init(topBottom: Styles.grid(3), leftRight: Styles.grid(3))
 
-    _ = self.amountAndRewardTitleStackView
-      |> \.axis .~ NSLayoutConstraint.Axis.vertical
-      |> \.translatesAutoresizingMaskIntoConstraints .~ false
-      |> \.isLayoutMarginsRelativeArrangement .~ true
+    _ = self.youreABackerLabel
+      |> \.font .~ .ksr_headline(size: 14)
+      |> \.text %~ { _ in Strings.Youre_a_backer() }
+      |> \.numberOfLines .~ 0
   }
 
   // MARK: - Binding
 
   override func bindViewModel() {
     super.bindViewModel()
 
-    self.pledgeCTAbutton.rac.title = self.vm.outputs.buttonTitleText
-    self.pledgeCTAbutton.rac.backgroundColor = self.vm.outputs.buttonBackgroundColor
     self.amountAndRewardTitleStackView.rac.hidden = self.vm.outputs.stackViewIsHidden
     self.amountOrRewardLabel.rac.text = self.vm.outputs.rewardTitle
+    self.pledgeButton.rac.backgroundColor = self.vm.outputs.buttonBackgroundColor
+    self.pledgeButton.rac.title = self.vm.outputs.buttonTitleText
+    self.spacer.rac.hidden = self.vm.outputs.spacerIsHidden
```

Comment:
We now also need to hide the spacer so we need 1 additional signal that should be tested as well